### DISCUSSION TITLE
Filter over date field [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2331] [Discord Threads]
> 🙋‍♂️ **Cris Acosta**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### tried over SQLite, following filter worked for me  
![image0](https://cdn.discordapp.com/attachments/982253968904314961/982272379931729940/Screenshot_2022-06-03_at_6.48.15_PM.png)  
#  
> 👨‍🦲 **dstala**  
#### dateColumn >= 2022-05-01   
#### AND dateColumn <= 2022-05-30  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/982253968904314961)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
